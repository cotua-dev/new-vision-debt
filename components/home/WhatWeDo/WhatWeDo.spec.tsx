import { render, RenderResult } from '@testing-library/react';
import { WhatWeDo } from './index';

describe('WhatWeDo Component', (): void => {
    let renderResult: RenderResult;

    beforeAll((): RenderResult => renderResult = render(<WhatWeDo/>));

    test('renders', (): void => {
        const { container } = renderResult;
        expect(container.firstChild).toMatchSnapshot();
    });

    afterAll((): void => renderResult.unmount());
});
