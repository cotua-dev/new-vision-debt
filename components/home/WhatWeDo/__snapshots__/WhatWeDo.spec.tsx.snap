// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WhatWeDo Component renders 1`] = `
<section
  class="py-12 px-2 flex"
>
  <div
    class="max-w-7xl mx-auto flex flex-col xl:flex-row items-center"
  >
    <div
      class="max-w-prose text-center mb-4 xl:text-left xl:max-w-lg"
    >
      <h2
        class="text-4xl text-daa-purple font-black mb-2"
      >
        Save Thousands Off Your Consumer Debts
      </h2>
      <p
        class="font-light"
      >
        Dealing with debts from credit cards, loans and medical bills can be a daunting process. We partnered with a network of nationwide attorneys that can negotiate and resolve your debts.
      </p>
    </div>
    <div
      class="flex flex-col xl:flex-row xl:flex-wrap justify-center xl:justify-end gap-4"
    >
      <div
        class="max-w-prose text-center xl:text-left xl:max-w-xs"
      >
        <strong
          class="text-2xl text-daa-purple font-black mb-2"
        >
          Cut Your Debt
        </strong>
        <p
          class="font-light"
        >
          Specialized debt litigation attorneys fight creditors so your unsecured debts are invalidated through the FDCPA (Fair Debt Collection Practices Act).
        </p>
      </div>
      <div
        class="max-w-prose text-center xl:text-left xl:max-w-xs"
      >
        <strong
          class="text-2xl text-daa-purple font-black mb-2"
        >
          Save Your Money
        </strong>
        <p
          class="font-light"
        >
          Why scramble each month to make your high-interest minimum payments. Start putting that money back in your pocket for future emergencies.
        </p>
      </div>
      <div
        class="max-w-prose text-center xl:text-left xl:max-w-xs"
      >
        <strong
          class="text-2xl text-daa-purple font-black mb-2"
        >
          Erase Bad Credit
        </strong>
        <p
          class="font-light"
        >
          Watch thousands of dollars in charge-offs & delinquencies disappear as they are disputed by our nationwide debt resolution legal team.
        </p>
      </div>
      <div
        class="max-w-prose text-center xl:text-left xl:max-w-xs"
      >
        <strong
          class="text-2xl text-daa-purple font-black mb-2"
        >
          Unsecured Loans
        </strong>
        <p
          class="font-light"
        >
          Get access to low-interest, unsecured loans. We shop multiple loan programs and banks to get you the lowest rates and terms.
        </p>
      </div>
    </div>
  </div>
</section>
`;
